FV3/conf/configure.fv3.intel_docker

```diff
@@ -0,0 +1,116 @@
+############
+# commands #
+############
+FC = mpif90
+CC = mpicc
+LD = $(FC) -cxxlib
+
+#########
+# flags #
+#########
+# default is 64-bit OpenMP non-hydrostatic build using AVX2
+DEBUG =
+REPRO =
+VERBOSE =
+OPENMP = Y
+AVX2 = Y
+HYDRO = N
+
+#NCEPLIBS_DIR=/myapps/ncep
+NEMSIOINC = -I$(NCEPLIBS_DIR)/include
+NCEPLIBS = -L$(NCEPLIBS_DIR)/lib -lbacio_4 -lsp_v2.0.2_d -lw3nco_d
+
+##############################################
+# Need to use at least GNU Make version 3.81 #
+##############################################
+need := 3.81
+ok := $(filter $(need),$(firstword $(sort $(MAKE_VERSION) $(need))))
+ifneq ($(need),$(ok))
+$(error Need at least make version $(need).  Load module gmake/3.81)
+endif
+
+#MKLROOT=/opt/intel/inteloneapi/mkl/latest
```

Comment:
Safe to use? Does GFDL also use it when they compile with Intel?

Reply:
Shouldn't be any problem as we wouldn't be importing the local MKL libraries.